kolibri/auth/api.py

```diff
@@ -268,4 +268,11 @@ def get_session(self, request):
 class KeepSessionAliveViewSet(viewsets.ViewSet):
 
     def retrieve(self, request, pk=None):
-        return Response()
+        user = get_user(request)
+        if isinstance(user, AnonymousUser):
+            user_id = None
+        else:
+            user_id = user.id
+        return Response({
+            'user_id': user_id,
+        })
```

Comment:
minor stylistic suggestion to avoid intermediary variable:

```python
if isinstance(user, AnonymousUser):
  return Response({ 'user_id': None })
else:
  return Response({ 'user_id': user.id })
```